### PR TITLE
#104: Show final picture in finished state

### PR DESCRIPTION
**Pull request by werdeil**
This is a first work to display final picture in the finished state:
Pros: 
- User without printer will see it before reaching the wait screen (useful if we use the "final image display time" setting)
- Landscape user will see a bigger version of the image (not fullscreen but quite bigger).

Cons:
- "Thanks" text is quite small and may be reduced if we want landscape picture to be bigger.

This needs to be tested more thoroughly before merging, with the forget mechanism and the final image display time setting.

### CODE REVIEW

**Comment by werdeil** [comment]
This is a first draft to address the request #104

File: pibooth/view/window.py [hunk 1]
```diff
@@ -60,14 +60,16 @@ def __init__(self, title,
                         (192, 0, 224, 0, 240, 0, 248, 0, 252, 0, 254, 0, 255, 0, 255,
                          128, 255, 192, 255, 224, 254, 0, 239, 0, 207, 0, 135, 128, 7, 128, 3, 0))
 
-    def _update_foreground(self, pil_image, pos=CENTER, resize=True):
+    def _update_foreground(self, pil_image, pos=CENTER, resize=True, fullscreen=False):
```
Comment (anxuae) [inline pibooth/view/window.py:63]: instead of adding a new parameter, you can use the 'pos' one.
pos=FULLSCREEN.

Comment (werdeil) [reply pibooth/view/window.py]: ok changed

**Comment by anxuae** [comment]
Other proposal is to display it during the "print" state:

- Print instruction + preview if a printer is installed
- Preview as big as possible if no printer installed

**Comment by anxuae** [comment]
I didn't find a better way.
@werdeil should we do it optional in the config?

**Comment by werdeil** [comment]
> I didn't find a better way.
> @werdeil should we do it optional in the config?

As you wish, if you don't want it to be by default yes let's add it in the options

**Comment by anxuae** [comment]
Renamed in config`finish_image_delay`-> `wait_image_delay `

Add in config `wait_image_delay`:

- `0` : only the "Thanks" screen is displayed
- `x` : fullscreen final picture is displayed during x seconds

**Review by anxuae — APPROVED** [review]

**Comment by werdeil** [comment]
I don't like the fact that in landscape the image takes the full height, I thought I had put margins did you change it @anxuae ?

**Comment by anxuae** [comment]
right, I will re-add it

**Comment by werdeil** [comment]
Ok with this two optional comments:
- maybe left and right people could be a little bit bigger with the landscape image
- maybe force the only thanks screen when the user press the forget option on the print screen.

**Pull request merged**